docs/source/conf.py

```diff
@@ -129,28 +130,33 @@
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
-html_theme = 'sphinx_rtd_theme'
+html_theme = 'furo'
 
 # Theme options are theme-specific and customize the look and feel of a theme
 # further.  For a list of options available for each theme, see the
 # documentation.
 html_theme_options = {
-    'style_external_links': True,
+    # 'style_external_links': True,  # Use after https://github.com/pradyunsg/furo/discussions/348
+    'navigation_with_keys': True,
+    'dark_css_variables': {'admonition-title-font-size': '0.95rem',
+                           'admonition-font-size': '0.92rem'},
+    'light_css_variables': {'admonition-title-font-size': '0.95rem',
+                            'admonition-font-size': '0.92rem'}
 }
 
 # Add any paths that contain custom themes here, relative to this directory.
 #html_theme_path = []
 
 # The name for this set of Sphinx documents.  If None, it defaults to
 # "<project> v<release> documentation".
-#html_title = None
+html_title = f"PTB v{version}"
 
 # A shorter title for the navigation bar.  Default is the same as html_title.
 #html_short_title = None
 
 # The name of an image file (relative to this directory) to place at the top
 # of the sidebar.
-html_logo = 'ptb-logo-orange.png'
+html_logo = 'ptb-logo_1024.png'
```

Comment:
maybe change the favicon to blue as well?

Reply:
there isn't one for blue, I'll add it